### DISCUSSION TITLE
Relative Mode not working in osu!lazer, but works in osu!stable

### DISCUSSION THREAD

**Discussion by qualk** [ppy/osu#17428] [Q&A]
Hello, it seems that relative mode isn't working in osu!lazer, but it works in osu!stable and everywhere else. 
I toggled raw input, mouse constriction to window, and other things, nothing worked.
Well, except disabling tablet. Absolute Mode worked, but, you know, I can't use the tablet 💀.

All OTD filters are turned off. Here is the OTD "Info" log:
```[Detect:Info]	Searching for tablets...
[Detect:Info]	Found tablet 'Wacom CTL-4100'
[Wacom CTL-4100:Info]	Output mode: Relative Mode
[Wacom CTL-4100:Info]	Relative Mode Sensitivity (X, Y): <20, 20>
[Wacom CTL-4100:Info]	Relative Mode Rotation: 0
[Wacom CTL-4100:Info]	Reset time: 00:00:00.1000000
[Wacom CTL-4100:Info]	Tip Binding: [Mouse Button Binding: Left]@0%
[Settings:Info]	Driver is enabled.
[Settings:Info]	Presets have been refreshed. Loaded 0 presets.
```

Here is the [runtime.log](https://github.com/ppy/osu/files/8339335/runtime.log).
I decided none of the other logs were necessary, please tell me if I'm wrong.

**Comment by peppy** [comment]
The question here is where is OTD getting the "relative mode" setting from.

**Reply by qualk** [reply]
ayo
Oh, maybe uploaded wrong log, I'll upload "Debug" log when I can.
But it was in Absolute Mode and it works (well, except for osu!laser obv).

**Reply by qualk** [reply]
I meant relative modem I'm so stupid 🤦🏾.
